go.mod

```diff
@@ -12,3 +19,4 @@ require (
 	github.com/stretchr/testify v1.6.1
 	golang.org/x/tools v0.0.0-20190614205625-5aca471b1d59
 )
+>>>>>>> upstream/master
```

Comment:
looks like you committed a file that had a conflict after a rebase.

Reply:
yeah, I thought I'd fixed that sunsequently?  Let me check again